packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -20,7 +20,10 @@ export default class RunPanelStore {
     @observable has_open_contract = false;
     @observable is_running = false;
     @observable is_drawer_open = true;
-
+    @observable is_restricted = false;
+    @observable max_losses = '';
+    @observable set_max_losses = '';
+    @observable run_limit = '';
```

Comment:
Should these values be integers rather than empty strings? Perhaps if they're not set we can use `null`. `''` suggests they will contain a string at some point.

Reply:
when using null onBotRunningEvent doesn't work fine :( 
and also I initialized these values same as the quick-strategy modal

Reply:
@mehrzad-fs can you create a store for `self-exclusion.jsx` rather than putting all the logic here